assemblies/docker/README.md

```diff
@@ -33,30 +33,63 @@ On macOS, an easy way to install `buildx` is to install [Docker Desktop Edge](ht
 ## Build
 
 Images are based on the Docker official [AdoptOpenJDK 11 JRE Hotspot](https://hub.docker.com/_/adoptopenjdk?tab=tags&page=1&name=11-jre-hotspot) image. If you want to
-build the Karaf image run:
+build the Karaf image you have the following choices:
 
-```
-sh build.sh
+1. Create the docker image from the SNAPSHOT version (from local distribution)
```

Comment:
That's not accurate: local distribution is not necessary SNAPSHOT. It could be a custom distribution locally built (like on CI). I would change this with:

`1. Create the docker image from a local distribution package`